client/analytics/components/report-chart/index.js

```diff
@@ -115,23 +84,41 @@ export class ReportChart extends Component {
 			};
 		} );
 
+		return chartData;
+	}
+
+	renderChart( mode, isRequesting, chartData ) {
+		const {
+			interactiveLegend,
+			itemsLabel,
+			legendPosition,
+			path,
+			query,
+			selectedChart,
+			showHeaderControls,
+			chartMode,
```

Comment:
I think this should have changed from `chartMode` to `mode`, otherwise it's always `undefined` for me.

Reply:
Good catch -- fixed to `mode`.